app/src/AddJoke.tsx

```diff
@@ -69,9 +81,11 @@ export default function AddJoke({ status = "empty" }: AddJokeProps) {
         Add Pun
       </button>
       <div className="jokeList">
-        {jokeList.map((joke: IJokes, key: number) => {
-          return <AppliedTheFavouriteJoke key={key} joke={joke} />;
-        })}
+        {jokeList.map(
```

Comment:
Sometimes VSCode auto formats lines of code when no changes have been made. If you want to avoid this, a handy tip is to to use the `Save without Formatting` setting